source/_integrations/vivotek.markdown

```diff
@@ -35,7 +35,7 @@ ip_address:
 name:
   description: This parameter allows you to override the name of your camera.
   required: false
-  default: Vivotek Camera
+  default: VIVOTEK Camera
```

Comment:
Is this also the default with capital letters?

Reply:
https://github.com/home-assistant/home-assistant/pull/31773